joern-cli/frontends/javasrc2cpg/src/test/scala/io/joern/javasrc2cpg/querying/CallTests.scala

```diff
@@ -218,34 +218,34 @@ class CallTests extends JavaSrcCodeToCpgFixture {
     val List(x) = cpg.call("add").l
     x.code shouldBe "this.add(argc, 3)"
     x.name shouldBe "add"
-    x.order shouldBe 2
+    x.order shouldBe 1
     x.methodFullName shouldBe "test.Foo.add:int(int,int)"
     x.signature shouldBe "int(int,int)"
-    x.argumentIndex shouldBe 2
+    x.argumentIndex shouldBe 1
     x.lineNumber shouldBe Some(9)
   }
 
   "should allow traversing from call to arguments" in {
     cpg.call("add").argument.size shouldBe 3
-    val List(arg0) = cpg.call("add").argument(0).l
+    val List(arg0) = cpg.call("add").receiver.l
     arg0.isInstanceOf[nodes.Identifier] shouldBe true
     arg0.asInstanceOf[nodes.Identifier].name shouldBe "this"
     arg0.code shouldBe "this"
-    arg0.order shouldBe 0
+//    arg0.order shouldBe 0
```

Comment:
Why is this commented out?

Reply:
Good question. I commented out a bunch of order tests while testing argument indices and have since fixed most of those, but it seems this and `thisParam` below slipped through. Will fix.